README.md

```diff
@@ -10,38 +10,39 @@ The technical specification of the data interface described in this document pro
 4. Function structure of the interface
 5. Types of communication with fiskaltrust.Middleware
 6. Operating categories
-Interface specification is provided by fiskaltrust.Interface nuget package, which can be found at https://www.nuget.org/packages/fiskaltrust.interface
+Interface specification is provided by fiskaltrust.Interface NuGet package, which can be found at https://www.nuget.org/packages/fiskaltrust.interface
 
 Official PDF documents that are built from the source in this repository can be found on the [Releases page](https://github.com/fiskaltrust/interface-doc/releases).
 
 ## Contributions
-If you want to contribute to this documentation, please review this README file to understand how it is structured and which tools are used.
+Please review this README file to understand how it is structured and which tools are used if you want to contribute to this documentation.
 
 ### Repository structure
 All documentation files are stored within the `/doc` folder in this repository (markdown, images, other static content).
 
-- One sub-directory and one or more markdown file(s) per chapter should be created (e.g. `general/general.md`.)
+- You should create one sub-directory and one or more markdown file(s) per chapter (e.g. `general/general.md`.)
   - 1st and 2nd level headings should be split into separate files.
-- Each chapter markdown file must be included into the table of contents in `/doc/toc.md`. DocFx will scan this file to compose the PDF documentation.
+- You must include each chapter markdown file in the table of contents in `/doc/toc.md`. DocFx will scan this file to compose the PDF documentation.
   - The order from the toc file is used to define the chapter order in the PDF file.
-  - You may also use multiple files per chapter, but keep in mind that a page-break is created by DocFx after every markdown file.
-- Images must be stored within a folder called `images`. Multiple image folders in different sub-directories are supported, DocFx just scans for the directory name.
+  - You may also use multiple files per chapter, but keep in mind that DocFx creates a page-break after every markdown file.
+- You must store images within a folder called `images`. Multiple image folders in different sub-directories are supported, DocFx scans for the directory name.
 
 #### References
 A reference to a markdown file can be created like this: `[Reference Text](./path-to-file/filename.md)`.
 > Note that reference paths should be relative.
 
-If you want to reference something specific inside a markdown file you need to create a anchor where you want your reference to point to.
+If you want to reference something specific inside a markdown file, you need to create an anchor where you want your reference to point to.
```

Comment:
..an anchor to which you want your reference to point.

Reply:
thank you, done!